vite.config.ts

```diff
@@ -84,7 +84,7 @@ export default defineConfig({
     port: 4000,
     proxy: {
       "/api": {
-        target: "https://careapi.ohc.network",
+        target: process.env.CARE_API ?? "https://careapi.ohc.network",
```

Comment:
Prefer import.meta.env over process.env

Reply:
@sainak that didn't work inside `vite.config.ts`

`CARE_API` should be a node level env than an app level env anyways right?

https://stackoverflow.com/a/66389044/7887936

<img width="685" alt="image" src="https://github.com/coronasafe/care_fe/assets/25143503/f9ba42aa-03c4-4208-b87c-0740889fed5d">

Reply:
Okay